Contributors.txt

```diff
@@ -18,13 +18,13 @@ Developers:
 	Thierry Bordaz
 	Sumit Bose
 	François Cami
+	Petr Čech
```

Comment:
When sorting, I use English collator:

```
collator = icu.Collator.createInstance(icu.Locale('en_US.UTF-8'))
authors.sort(cmp=collator.compare, key=lambda x: x.split()[-1].lower())
```

I could change it to Czech collator, but then for example surnames beginning with "Ch" are sorted after "H", which would confuse English speakers.

So I would rather stick with current working version (already applied to names like Špaček) to make the sorting clear for wide population.